src/components/navbar/Navbar.css

```diff
@@ -1,38 +1,44 @@
-.logo {
-  width: 4rem;
-  height: 4rem;
-  margin: 0 1rem;
-}
-
-.nav-search {
-  padding: 0.2rem;
-}
+@import "../../App.css";
```

Comment:
u dont need to import app.css explicitly in each component file it gets passed down through the hierarchy anyway